assets/js/modules/pagespeed-insights/datastore/constants.js

```diff
@@ -17,3 +17,12 @@
  */
 
 export const STORE_NAME = 'modules/pagespeed-insights';
+
+// Form ID for PageSpeed widget.
+export const FORM_DASH_WIDGET = 'pagespeedWidget';
```

Comment:
May be worth making this more specific, to avoid potential conflicts, maybe `pagespeedDashboardWidget`? Or `webVitalsDashboardWidget`?